codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -493,10 +515,18 @@ class FluentClientGenerator(
                         /// Consumes this builder, creating a customizable operation that can be modified before being
                         /// sent.
                         // TODO(enableNewSmithyRuntime): Remove `async` and `Result` once we switch to orchestrator
-                        pub async fn customize(self) -> #{Result}<
-                            #{CustomizableOperation},
-                            #{SdkError}<#{OperationError}>
-                        > {
+                        pub async fn customize(
+                            self,
+                        ) -> #{Result}<
+                            #{CustomizableOperation}<
+                                #{Box}<dyn #{CustomizableSend}<
```

Comment:
Is there a way we can hide the `Box<dyn CustomizableSend` piece from the public API to maximize flexibility in refactoring this in the future?

Reply:
Maybe add yet another alias `BoxedCustomizableSend` ?

Reply:
An alias won't necessarily let us change the type later. If we're using a `Box<dyn>` as the generic, then it seems like maybe it doesn't need to be a generic in the first place. Perhaps `CustomizableOperation` only takes the output and error types as generic args.

Reply:
Ah, I see what you're saying; Just expose the output type & the error type and hide the function part of the type parameter, `S`,  from `CustomizableOperation`. Let me try making that change.